cinema/permissions.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
There is a syntax error in the import statement. Replace the forward slash with a comma to separate the imported modules: `from rest_framework.permissions import BasePermission, SAFE_METHODS`.